src/ontology/pato-edit.obo

```diff
@@ -22249,6 +22249,116 @@ comment: This is distinct from cortical bipolar morphology which has two dendrit
 is_a: PATO:0010006 ! cell morphology
 property_value: http://purl.org/dc/terms/contributor https://orcid.org/0000-0001-7258-9596
 
+[Term]
+id: PATO:0070044
+name: cellular electrophysiological quality
+def: "A cellular quality that exists by virtue of the electrical properties of the cell." [WikipediaVersioned:Electrophysiology&oldid=1059385774]
+xref: ilxtr:ElectrophysiologicalPhenotype
+is_a: PATO:0001396 ! cellular quality
+
+[Term]
+id: PATO:0070045
+name: spiking phenotype
+xref: ilxtr:SpikingPhenotype
+is_a: PATO:0070044 ! cellular electrophysiological quality
+
+[Term]
+id: PATO:0070046
+name: regular spiking non pyramidal phenotype
+xref: ilxtr:RegularSpikingNonPyramidalPhenotype
+is_a: PATO:0070045 ! spiking phenotype
+disjoint_from: PATO:0070059 ! fast spiking phenotype
+
+[Term]
+id: PATO:0070047
+name: Petilla sustained spiking phenotype
```

Comment:
we don't normally include the name of the terminology system in the term label, unless really necessary for disambiguation.

Why not just "sustained spiking phenotype"

Also should all Petilla terms not have some kind of provenance to https://www.nature.com/articles/nrn2402

(I realize this is a WIP apologies if this is all part of your plan already)

Reply:
was hoping provenance would be shown by an xref to that paper in def when that is done. Will also add that xref to the Petilla synonyms.